source/api/commands/writefile.md

```diff
@@ -54,6 +53,12 @@ Pass in an options object to change the default behavior of `cy.writeFile()`.
 Option | Default | Description
 --- | --- | ---
 `log` | `true` | {% usage_options log %}
+`flag` | `w` | File system flag as used with {% url `fs.writeFile` https://nodejs.org/api/fs.html#fs_file_system_flags %}
+`encoding` | `utf8` | The encoding to be used when writing to the file
+
+{% note info %}
+To use encoding with other options, have your options object be your third parameter and include encoding there. This is the same behavior as`fs.writeFile`.
```

Comment:
missing space in between as`fs.writeFile`, also it is good and encouraged to link to fs docs if you want.